worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/FastSchematicReaderV3.java

```diff
@@ -136,6 +139,27 @@ public Clipboard read(final UUID uuid, final Function<BlockVector3, Clipboard> c
                     this.dataVersion = this.dataInputStream.readInt();
                     this.dataFixer = ReaderUtil.getVersionedDataFixer(this.dataVersion, platform, platform.getDataVersion());
                 }
+                case "Metadata" -> {
+                    LinCompoundTag metadataCompoundTag =
```

Comment:
I would prefer the data here to be read in a similar way like the other parts of the reader. So, not reading a whole compound in memory but streaming over the tags - especially as the metadata might be huge, depending where the schematic was generated.

Reply:
@PierreSchwang Unfortunately, I had already made an attempt to preserve the performance oriented goals of the stream reading. As it turns out, I am lacking in familiarity on a more consist approach, hence my hasted attempt to get it working properly.

Any help would be appreciated!